### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by danmanj**
More cleanup.

### CODE REVIEW

File: distbench_utils.cc [hunk 1]
```diff
@@ -82,8 +82,15 @@ void InitLibs(const char* argv0) {
...
 }
 
-absl::StatusOr<DeviceIpAddress> IpAddressForDevice(std::string_view netdev) {
-  return GetBestAddress(use_ipv4_first, netdev);
+absl::StatusOr<DeviceIpAddress> IpAddressForDevice(std::string_view netdev,
+                                                   int ip_version) {
+  if (ip_version == 4) {
+    return GetBestAddress(true, netdev);
+  } else if (ip_version == 6) {
+    return GetBestAddress(false, netdev);
```
Comment (oserres) [inline distbench_utils.cc:90]: For the case we request a specific version, shouldn't we check that we only get the version requested and fail if not ?

**Review by oserres — APPROVED** [review]
LGTM

File: distbench_utils.cc (hunk 1 above)
Comment (danmanj) [reply distbench_utils.cc:90]: Yes. We just fail in the caller (homa) for now.

Comment (danmanj) [reply distbench_utils.cc:90]: fixing this now, will add one more commit before pushing.

**Review by AbhayPatil — APPROVED** [review]

**Comment by oserres** [comment]
LGTM (including last 2 commits)

**Pull request merged**